bpf/lib/common.h

```diff
@@ -560,7 +560,7 @@ struct lb6_service_v2 {
 	__u32 backend_id;
 	__u16 count;
 	__u16 rev_nat_index;
-	__u16 weight;
+	__u16 weight;		/* Currently unused. */
```

Comment:
Nit: same comment for `lb4_service_v2`?

Reply:
We have all the comments in `lb4_service_v2`, so I removed the above snippet.